protocol/grpc/server.go

```diff
@@ -134,18 +136,12 @@ func waitGrpcExporter(providerServices map[string]*config.ServiceConfig) {
 func registerService(providerServices map[string]*config.ServiceConfig, server *grpc.Server) {
 	for key, providerService := range providerServices {
 		service := config.GetProviderService(key)
-
 		ds, ok := service.(DubboGrpcService)
 		if !ok {
```

Comment:
Is 'ds' may be nil ?

Reply:
> Is 'service' may be nil ?

Reply:
`service` check exist: https://github.com/apache/dubbo-go/blob/master/config/config_loader.go#L259-L264.